packages/twenty-front/src/modules/object-record/record-index/components/RecordIndexBoardContainerEffect.tsx

```diff
@@ -33,6 +34,15 @@ export const RecordIndexBoardContainerEffect = ({
     getOnFetchMoreVisibilityChangeState,
   } = useRecordBoard(recordBoardId);
 
+  const { currentViewSelector } = useViewScopedStates({
+    viewScopeId: viewBarId,
```

Comment:
this is used that way in other places in the code. Should I do a refacto?

Reply:
No, we'll take care of that while refactoring the ViewBar later this month